Makefile

```diff
@@ -10,3 +10,13 @@ DOCKER_BUILD_CONTEXT = ..
 .PHONY: $(shell test -f common/common.mk || echo >&2 'Please do "git submodule update --init" first.')
 
 include common/common.mk
+
+.PHOHY: generate-new
+generate-new:
+	for version in ${VERSIONS} ; do \
+		if [[ "$$version" == *-minimal ]]; then \
+			python3 ./generator.py -v $$version -m manifest-minimal.yml -s specs/multispec.yml ; \
```

Comment:
Why not add a shebang to `generator.py` instead?

Reply:
How is shebang better?

Reply:
The fact the script is in Python is an irrelevant implementation detail in this case, thus shebang makes sense IMHO.